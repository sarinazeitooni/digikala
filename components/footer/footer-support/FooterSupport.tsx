import React, {useState} from 'react';
import style from './styles/footer-support.module.scss';
import ExpandLessIcon from '@material-ui/icons/ExpandLess';
import messages from './messages/messages'
const FooterSupport = ()=>{
    const [features , SetFeatures] = useState(messages.features);
    return(
        <div className={style['footer-support-container']}>
            <div className={style['back-to-top-container']}>
                <img src='https://www.digikala.com/static/files/2a4774d7.svg'/>
                <button className={style['back-to-top']}>{messages.backToTop} <span><ExpandLessIcon/></span></button>
            </div>
            <div className={style['support-text']}>{messages.support}</div>
            <div className={style['footer-features-container']}>
                {features.map((item)=>{
                    return(
                        <div className={style['feature-item-container']}>
                            <img src={item.icon}/>
                            <div>{item.text}</div>
                        </div>
                    )
                })}
            </div>
        </div>
    )
}
export default FooterSupport;