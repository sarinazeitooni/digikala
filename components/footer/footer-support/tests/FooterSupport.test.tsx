import {shallow} from "enzyme";
import FooterSupport from "../FooterSupport";
describe("FooterSupport component" , ()=>{
    it("should render correctly" , ()=>{
        const container = shallow(<FooterSupport/>);
        expect(container).toMatchSnapshot();
    })
});