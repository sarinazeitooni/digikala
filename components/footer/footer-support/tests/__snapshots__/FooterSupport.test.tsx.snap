// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FooterSupport component should render correctly 1`] = `ShallowWrapper {}`;
